news/2025/2025-04-08-beatmap-spotlights-revival-spring-2025.md

```diff
@@ -0,0 +1,263 @@
+---
+layout: post
+title: "Beatmap Spotlights Revival: Spring 2025"
+date: 2025-04-09 01:00:00 +0000
+---
+
+Haven't heard this name for a while? Check out all the major changes to Beatmap Spotlights! This time, we are also seeking fresh blood to ensure the project can run smoothly for many seasons to come!
+
+![](/wiki/shared/news/2025-04-08-beatmap-spotlights-revival-spring-2025/banner.jpg)
+
+It's been some time since we last met here indeed... let's examine what happened in the meantime, since there's certainly a lot to cover.
```

Comment:
```suggestion
It's been some time since we last met here indeed... Let's examine what happened in the meantime, since there's certainly a lot to cover.
```